packages/mendel-development-loader/loader.js

```diff
@@ -14,7 +14,7 @@ function MendelLoaderDev(existingVariations, config, parentModule) {
     this._existingVariations = existingVariations;
     this._config = config;
     this._parentModule = parentModule || module.parent;
-    this._ssrReady = isValidDir(config.serveroutdir);
```

Comment:
How about instantiate as `false` first? Looks like the new function can return undefined for a while?